addons/account_edi_ubl_cii/data/ubl_20_templates.xml

```diff
@@ -408,4 +429,12 @@
         </CreditNote>
     </template>
 
+    <template id="ubl_20_DebitNote">
+        <DebitNote xmlns="urn:oasis:names:specification:ubl:schema:xsd:DebitNote-2"
+                   xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+                   xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+            <t t-call="{{InvoiceType_template}}"/>
+        </DebitNote>
+    </template>
```

Comment:
I checked the difference between DebitNoteType and InvoiceType in UBL 2.0: 
`set(nodes_debit_note_type) - set(nodes_invoice_type) = {'RequestedMonetaryTotal', 'DebitNoteLine', 'DiscrepancyResponse'}`. However, I would create a template `ubl_20_DebitNoteType` by inheriting `ubl_20_InvoiceType` with primary=True and you would simply add these 3 nodes (if they are used) at the right place. This way, we avoid adding `DebitNoteLine` inside the Invoice template.

Reply:
As talked about, since CreditNote does the same thing, I tried to keep it the same way. We can discuss with Laurent, but since there probably won't be much other document types this has the least impact in changes and is still fairly readable.

Reply:
So, this has been refactored as discussed with Laurent. One `CommonType` and the other 3 inheriting from that.